src/test/chain-simulator/accounts.cs-e2e.ts

```diff
@@ -1187,4 +1191,346 @@ describe('Accounts e2e tests with chain simulator', () => {
       expect(response.data).toBeGreaterThanOrEqual(1);
     });
   });
+
+  describe('GET /accounts/:address/transfers', () => {
+    it('should return transfers for a given address', async () => {
+      const countResponse = await axios.get(`${config.apiServiceUrl}/accounts/${config.aliceAddress}/transfers/count`);
+      const expectedCount = countResponse.data;
+
+      const response = await axios.get(`${config.apiServiceUrl}/accounts/${config.aliceAddress}/transfers`);
+      expect(response.status).toBe(200);
+      expect(response.data.length).toStrictEqual(expectedCount);
+    });
+
+    it('should support pagination', async () => {
+      const response = await axios.get(`${config.apiServiceUrl}/accounts/${config.aliceAddress}/transfers?size=2`);
+      expect(response.status).toBe(200);
+      expect(response.data.length).toStrictEqual(2);
+    });
+
+    it('should return transfers with sender parameter', async () => {
+      const countResponse = await axios.get(`${config.apiServiceUrl}/accounts/${config.aliceAddress}/transfers/count?sender=${config.aliceAddress}`);
+      const expectedCount = countResponse.data;
+
+      const response = await axios.get(`${config.apiServiceUrl}/accounts/${config.aliceAddress}/transfers?sender=${config.aliceAddress}`);
+      expect(response.status).toBe(200);
+      expect(response.data.length).toStrictEqual(expectedCount);
+
+      for (const transfer of response.data) {
+        expect(transfer.sender).toBe(config.aliceAddress);
+      }
+    });
+
+    it('should return transfers with receiver parameter', async () => {
+      const receiverAddress = 'erd1qqqqqqqqqqqqqqqpqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqzllls8a5w6u';
+      const response = await axios.get(`${config.apiServiceUrl}/accounts/${config.aliceAddress}/transfers?receiver=${receiverAddress}`);
+      expect(response.status).toBe(200);
+      expect(response.data.length).toBeGreaterThanOrEqual(1);
+
+      for (const transfer of response.data) {
+        expect(transfer.receiver).toBe(receiverAddress);
+      }
+    });
+
+    it('should return transfers with token parameter', async () => {
+      const accountTokens = await axios.get(`${config.apiServiceUrl}/accounts/${config.aliceAddress}/tokens`);
+      const token = accountTokens.data[0].identifier;
+
+      const countResponse = await axios.get(`${config.apiServiceUrl}/accounts/${config.aliceAddress}/transfers/count?token=${token}`);
+      const expectedCount = countResponse.data;
+
+      const response = await axios.get(`${config.apiServiceUrl}/accounts/${config.aliceAddress}/transfers?token=${token}`);
```

Comment:
```suggestion
      const endpoint = `{config.apiServiceUrl}/accounts/${config.aliceAddress}/transfers`
      const countResponse = await axios.get(`${endpoint}/count?token=${token}`);
      const expectedCount = countResponse.data;

      const response = await axios.get(`${endpoint}?token=${token}`);
```

easier to follow and less errors

Reply:
what errors?
it's clear that we have `countResponse` that returns a number ( `count` ) and that assign the value to the expectedCount value.